en/docs/reference/connectors/email-connector/1.x/email-connector-1.x-config.md

```diff
@@ -0,0 +1,687 @@
+        </tr>
+        <tr>
+            <td>name</td>
+            <td>Unique name the connection is identified by.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>username</td>
+            <td>Username used to connect with the mail server.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>password</td>
+            <td>Password to connect with the mail server.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>connectionType</td>
+            <td>Email connection type (protocol) that should be used to establish the connection with the server. (IMAP/IMAPS/POP3/POP3S/SMTP/SMTPS).</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>readTimeout</td>
+            <td>The socket read timeout value. E.g., 100000.</td>
+            <td>Optional</td>
+        </tr>
+        <tr>
+            <td>connectionTimeout</td>
+            <td>The socket connection timeout value. E.g., 100000.</td>
+            <td>Optional</td>
+        </tr>
+        <tr>
+            <td>writeTimeout</td>
+            <td>The socket write timeout value. E.g., 100000.</td>
+            <td>Optional</td>
+        </tr>
+        <tr>
+            <td>requireTLS</td>
+            <td>Whether the connection should be established using TLS. The default value is false. Therefore, for secured protocols SSL will be used by default.</td>
+            <td>Optional</td>
+        </tr>
+        <tr>
+            <td>checkServerIdentity</td>
+            <td>Whether server identity should be checked.</td>
+            <td>Optional</td>
+        </tr>
+        <tr>
+            <td>trustedHosts</td>
+            <td>Comma separated string of trust host names.</td>
+            <td>Optional</td>
+        </tr>
+        <tr>
+            <td>sslProtocols</td>
+            <td>Comma separated string of SSL protocols.</td>
+            <td>Optional</td>
+        </tr>
+        <tr>
+            <td>cipherSuites</td>
+            <td>Comma separated string of Cipher Suites.</td>
+            <td>Optional</td>
+        </tr>
+        <tr>
+            <td>maxActiveConnections</td>
+            <td>Maximum number of active connections in the pool. When negative, there is no limit to the number of objects that can be managed by the pool at one time. Default is 8.</td>
+            <td>Optional</td>
+        </tr>
+        <tr>
+            <td>maxIdleConnections</td>
+            <td>Maximum number of idle connections in the pool. When negative, there is no limit to the number of objects that may be idle at one time. Default is 8.</td>
+            <td>Optional</td>
+        </tr>
+        <tr>
+            <td>maxWaitTime</td>
+            <td>Specifies the number of milliseconds to wait for a pooled component to become available when the pool is exhausted and the exhaustedAction is set to WHEN_EXHAUSTED_WAIT. If maxWait is negative, it will be blocked indefinitely. Default is -1.</td>
```

Comment:
```suggestion
            <td>Specifies the number of milliseconds to wait for a pooled component to become available when the pool is exhausted and the exhaustedAction is set to <code>WHEN_EXHAUSTED_WAIT</code>. If <code>maxWait</code> is negative, it will be blocked indefinitely. Default is <code>-1</code>.</td>
```